mne/datasets/sleep_physionet/_utils.py

```diff
@@ -115,16 +114,12 @@ def _update_sleep_temazepam_records(fname=TEMAZEPAM_SLEEP_RECORDS):
     data = data.set_index(("Subject - age - sex", "Nr"))
     data.index.name = "subject"
     data.columns.names = [None, None]
-    kwargs = dict()
-    # TODO VERSION can be removed once we require Pandas 2.1
-    if "future_stack" in inspect.getfullargspec(pd.DataFrame.stack).args:
-        kwargs["future_stack"] = True
     data = (
         data.set_index(
             [("Subject - age - sex", "Age"), ("Subject - age - sex", "M1/F2")],
             append=True,
         )
-        .stack(level=0, **kwargs)
+        .stack(level=0, future_stack=True)
```

Comment:
```suggestion
        # TODO VERSION future_stack goes away w/ Pandas 3.0
        .stack(level=0, future_stack=True)
```

Reply:
I was going to say maybe we leave this as-is then... but we'll get a loud error from `pip-pre` once 3.0 dev cycle starts and we'll probably have other stuff to fix then anyway